src/generators/kotlin/KotlinConstrainer.ts

```diff
@@ -0,0 +1,142 @@
+import { ConstrainedEnumValueModel } from 'models';
```

Comment:
```suggestion
import { ConstrainedEnumValueModel } from '../../models';
```

This is why the black-box tests are complaining 🙂